README.md

```diff
@@ -23,13 +23,14 @@ Run `make` to build `./bin/cloudwatch.so`. Then use with Fluent Bit:
 ### Plugin Options
 
 * `region`: The AWS region.
-* `log_group_name`: The name of the CloudWatch Log Group that you want log records sent to.
-* `log_stream_name`: The name of the CloudWatch Log Stream that you want log records sent to.
-* `log_stream_prefix`: Prefix for the Log Stream name. The tag is appended to the prefix to construct the full log stream name. Not compatible with the `log_stream_name` option.  
+* `log_group_name`: The name of the CloudWatch Log Group that you want log records sent to. This value allows a template in the form of `${variable}`. See `log_stream_name` description for more. The app will attempt to create missing log groups, and will throw an error if it does not have access.
```

Comment:
Shouldn't it be `$(variable)`? From #78 conversation, we planned to replace `${ }` with `$( )`. Am I missing something?

Reply:
I was waiting for your team to provide feedback on the chosen variable identifiers. If y'all are good with `#()` I'll go ahead and fix the doc. Is there any additional feedback before making a final commit?

Reply:
We voted and we prefer `$()`

Reply:
Sounds good, I'll switch it up and test before committing.

Reply:
Added a commit changes the identifiers. Updated the readme. Made some comments better. Fixed a few log lines and added a couple debug lines. If you guys don't mind I'll follow-up with a clean-up PR (no features).